pages/osx/lp.md

```diff
@@ -0,0 +1,8 @@
+#lp
+> print files
+- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
+'lp -d {{device_name}} {{path/to/filename}}
```

Comment:
Hi, he meant you used the `'` (single quote) around commands, but should use a backtick instead (`).